checkov/sca_package/output.py

```diff
@@ -50,7 +50,55 @@ def output_row(self) -> List[str]:
         ]
 
 
-def create_report_record(
+def create_report_license_record(
+    rootless_file_path: str,
+    file_abs_path: str,
+    check_class: str,
+    licenses_status: _LicenseStatus,
+) -> Record:
+    package_name = licenses_status["package_name"]
+    package_version = licenses_status["package_version"]
+
+    check_result: _CheckResult = {
+        "result": CheckResult.FAILED,
+    }
+
+    policy = licenses_status["policy"]
+    status = licenses_status["status"]
+
+    if status == "COMPLIANT":
+        check_result = {
+            "result": CheckResult.PASSED,
+        }
```

Comment:
```suggestion
        check_result["result"] = CheckResult.PASSED
```
no need to create a new dict 🙂

Reply:
will be solved in next PR